pkg/beacon/entry/submission.go

```diff
@@ -99,11 +83,21 @@ func (res *relayEntrySubmitter) submitRelayEntry(
 				}
 			}
 
-			return entryErr
+			logger.Infof(
+				"[member:%v] successfully fired relay entry "+
+					"submission at block: [%v]",
```

Comment:
Maybe "successfully submitted relay entry transaction to the mempool at block ..."?

Reply:
See 364b53a7